### PR TITLE
Refactor entity.attribute mref --> one-to-many

### PR DESCRIPTION
**Pull request by dennishendriksen**
#### Checklist
- [ ] Unit tests
- [ ] Updated testplan

### CODE REVIEW

File: molgenis-data/src/main/java/org/molgenis/data/meta/model/Attribute.java [hunk 1]
```diff
@@ -135,6 +138,76 @@ public Attribute setName(String name)
...
+		return getInt(SEQUENCE_NR);
+	}
+
+	public Attribute setSequenceNumber(int seqNr)
+	{
+		set(SEQUENCE_NR, seqNr);
+		return this;
+	}
+	
+	// FIXME rename to getEntityType
```
Comment (tommydeboer) [inline molgenis-data/src/main/java/org/molgenis/data/meta/model/Attribute.java]: Can you add sticky on the board or do this refactoring?

File: molgenis-data/src/main/java/org/molgenis/data/meta/model/Attribute.java [hunk 2]
```diff
@@ -135,6 +138,76 @@ public Attribute setName(String name)
...
+	public EntityType getEntity()
+	{
+		return getEntity(ENTITY, EntityType.class);
+	}
+
+	// FIXME rename to setEntityType
```
Comment (tommydeboer) [inline molgenis-data/src/main/java/org/molgenis/data/meta/model/Attribute.java]: Can you add sticky on the board or do this refactoring?

File: molgenis-data/src/main/java/org/molgenis/data/meta/model/Attribute.java [hunk 3]
```diff
@@ -224,15 +297,10 @@ public Attribute setDataType(AttributeType dataType)
 	 *
 	 * @return Iterable of attributes or empty Iterable if no attribute parts exist
 	 */
+	// FIXME rename to getChildren
```
Comment (tommydeboer) [inline molgenis-data/src/main/java/org/molgenis/data/meta/model/Attribute.java]: Can you add sticky on the board or do this refactoring?

**Comment by tommydeboer** [comment]
Removing one of the attributes of an EntityType gives:
Error! entity [sys_md_Attribute] with id [AAAACV5564YDKGWXPB5UYUAAAI] does not exist

**Comment by tommydeboer** [comment]
When removing a compound attribute from an EntityType:

Error! Could not commit JDBC transaction; nested exception is org.postgresql.util.PSQLException: ERROR: update or delete on table "sys_md_Attribute" violates foreign key constraint "sys_md_Attribute_parent_fkey" on table "sys_md_Attribute" Detail: Key (id)=(AAAACV5564YDGGWXPB5UYUAAAM) is still referenced from table "sys_md_Attribute".

**Comment by dennishendriksen** [comment]
See meta model refactoring pull request

**Pull request closed without merging**